executor/executor_bsd.h

```diff
@@ -66,6 +66,13 @@ static void cover_open(cover_t* cov, bool extra)
 	unsigned long cover_size = kCoverSize;
 	if (ioctl(cov->fd, KIOSETBUFSIZE, &cover_size))
 		fail("ioctl init trace write failed");
+	if (extra) {
+		struct kio_remote_attach args;
```

Comment:
How do you feel about designated initializers? I believe you can save a line and be certain the rest of the fields won't end up uninitialized:
```
struct kio_remote_attach args = {
    .subsystem = KCOV_REMOTE_COMMON,
    .id = 0  // This should be not absolutely necessary unless you want to be explicit.
};
```

Reply:
Sure, I was just matching the NetBSD chunk. We should probably change both for consistency.